task-requests/details/script.js

```diff
@@ -23,6 +24,23 @@ const errorMessage =
   'The requested operation could not be completed. Please try again later.';
 let taskId;
 
+async function getSelfUser() {
+  const res = await fetch(`${API_BASE_URL}/users/self`, {
+    method: 'GET',
+    credentials: 'include',
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
+
+  const self_user = await res.json();
+  return self_user;
+}
```

Comment:
- i think a function to call `users/self` is already present as it is called after the user logs in, see if you can use it.

Reply:
To use that predefined function i have to either include the script file in the index.js file or make the script type module to import that function.

Reply:
Make it a module then as to call/self a function is already there

Reply:
we have to get that function like this if we make it moudle 
```javascript 
import { getIsSuperUser } from '../../applications/utils.js';
```
or we can just add this in the html head and not convert the file into module
```html
 <script src="/utils.js"></script>